doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+TODO: add highlights
+
+This release includes
+
+Backwards incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: add examples of how to deal with these changes
+
+-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Refactor regridding (:pull:`2231`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `linear_extrapolate` in recipe checks (:pull:`2324`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `unstructured_nearest` in recipe checks (:pull:`2336`) by :user:`schlunma`
+
+Bug fixes
+~~~~~~~~~
+
+-  Fixing missing height 2m coordinates in GFDL-CM4 and KIOST-ESM (:pull:`2294`) by :user:`Karen-A-Garcia`
+-  Do not overwrite facets from recipe with CMOR table facets for derived variables (:pull:`2255`) by :user:`bouweandela`
+-  Fix error message in variable definition check (:pull:`2313`) by :user:`enekomartinmartinez`
+-  Unify dtype handling of preprocessors (:pull:`2393`) by :user:`schlunma`
+
+CMOR standard
+~~~~~~~~~~~~~
+
+-  Also read default custom CMOR tables if custom location is specified (:pull:`2279`) by :user:`schlunma`
+-  Add custom CMOR table for total cloud water (tcw) (:pull:`2277`) by :user:`axel-lauer`
+-  Add height for sfcWindmax in MPI HighRes models (:pull:`2292`) by :user:`malininae`
+-  Fixed 'positive' attribute in custom rtnt table (:pull:`2367`) by :user:`schlunma`
+-  Fix `positive` attributes in custom CMOR variables (:pull:`2380`) by :user:`schlunma`
+-  Log CMOR check and generic fix output to separate file (:pull:`2361`) by :user:`schlunma`
+
+Computational performance improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  More lazy fixes and preprocessing functions (:pull:`2325`) by :user:`bouweandela`
+-  Made preprocessors mask_landsea, mask_landseaice and mask_glaciated lazy  (:pull:`2268`) by :user:`joergbenke`
+-  Use lazy weights for `climate_statistics` and `axis_statistics` (:pull:`2346`) by :user:`schlunma`
+-  Fixed potential memory leak in `local_solar_time` (:pull:`2356`) by :user:`schlunma`
+-  Implement lazy area weights (:pull:`2354`) by :user:`schlunma`
+-  Avoid large chunks in `climate_statistics` preprocessor function with `period='full'` (:pull:`2404`) by :user:`bouweandela`
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Add highlights to the v2.10 changelog (:pull:`2282`) by :user:`bouweandela`
```

Comment:
That probably already werkt into the v2.10 release?

Reply:
We made some minor local changes to the `draft_release_notes.py` script and re-ran.  Changes are now in: 63619a0e2.

We will open a new issue to improve this permanently.

```diff
--- a/esmvaltool/utils/draft_release_notes.py
+++ b/esmvaltool/utils/draft_release_notes.py
@@ -44,7 +44,7 @@ GITHUB_REPO = {
 TIMEZONE = ZoneInfo("CET")
 
 PREVIOUS_RELEASE = {
-    'esmvalcore': datetime.datetime(2023, 6, 6, 0, tzinfo=TIMEZONE),
+    'esmvalcore': datetime.datetime(2023, 12, 19, 18, tzinfo=TIMEZONE),
     'esmvaltool': datetime.datetime(2023, 6, 20, 0, tzinfo=TIMEZONE),
 }
 
@@ -62,11 +62,11 @@ LABELS = {
         'documentation',
         'fix for dataset',
         'installation',
-        'iris',
+        # 'iris',
         'preprocessor',
-        'release',
+        # 'release',
         'testing',
-        'UX',
+        # 'UX',
         'variable derivation',
         'enhancement',  # uncategorized, keep at the bottom
     ),
```

Reply:
We also removed this PR which was from v.2.10.0 as well: 92ec5c30a